src/data/projects.json

```diff
@@ -181,14 +181,9 @@
         "narrative": "<p>Unlike other education platforms, <a href='https://www.oppiafoundation.org/about'>Oppia</a> is heavily volunteer-based, from the people who create the lessons to the developers who build new features. Because of this, it is structured as a strong network of open source developers, artists, teachers, and students. From day 1, as a part of the Oppia team, you will be welcomed by an onboarding team at Oppia, which will help you through an initial starter project.</p><p>In previous years, students who worked with Oppia have developed new features like Takeout, which lets users manage and delete their own data, and a Docker container to allow Oppia's Windows developers to have the same convenience as Linux and macOS developers. As an open source developer for Oppia, you will be interacting with programmers who have the same motivations as you all across the world, from India to Malaysia to the US.</p><p>Based on AngularJS and Python, Oppia is constantly improving its platform through direct feedback from the students, teachers, and volunteers. Oppia regularly tests its lessons with students around the world, and they conduct randomized trials to ensure their platform is effective. Therefore, you can rest assured that your contributions to Oppia are actually supporting students' education around the world. Examples of future projects that you can work on include creating features to help teachers craft effective lessons, developing Oppia's Android application, improving the lesson feedback pipeline from the user to the teacher, and developing Oppia's testing and quality assurance infrastructure to make sure the platform is reliable for students.</p><p>If you would like to learn more, check out Oppia's <a href='https://github.com/oppia/oppia'>GitHub</a> and <a href='https://oppia.org'>website</a>.",
         "organization": "Oppia is an online education platform that serves almost 430,000 students worldwide. ",
         "summary": "We support high-quality online learning experiences for students with limited access to formal education.",
-        "team": [
```

Comment:
are we sure we want to delete everyone besides leadership from the people page for now? I guess we could add people back as the year goes on but the page looks a little sad and empty right now + I bet some of these people will be continuing from last year. what do you think?

Reply:
It probably won't hurt to keep previous members there for now while we wait for the new teams to populate winter quarter, so I'll just change the leadership for the time being.